modules/odc-creating-functions.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-functions.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-functions_{context}"]
+= Creating a function in the web console
+
+You can create a function from a Git repository by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have logged in to the {product-title} web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created or have access to a Git repository that contains the code for your function. The repostory must contain a `func.yaml` file or use the `s2i` build strategy.
```

Comment:
FYI, in that case the text in the UI is incorrect, because it states "or":
![image](https://user-images.githubusercontent.com/40172997/236244196-ca1332ae-0592-45ea-9d44-2491a02fc6e3.png)

probably need to open a bug with the ODC team to fix this

Reply:
Created a bug to fix this: https://issues.redhat.com/browse/OCPBUGS-13120